pom.xml

```diff
@@ -22,10 +22,19 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.12</version>
+            <version>4.13.2</version>
             <scope>test</scope>
         </dependency>
-    </dependencies>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>2.17.2</version>
+        </dependency>
```

Comment:
why do you need this dependency?

Reply:
not fixed :)

@torbinofficial  I'll approve it, but please try to avoid adding unnecessary dependencies in future :)